static/js/components/dashboard/CourseAction.js

```diff
@@ -87,55 +89,83 @@ export default class CourseAction extends React.Component {
     );
   }
 
+  renderDescription(text: string, className: string): React$Element<*> {
+    return text.length > 0 ?
+      <div className={className}>{text}</div> :
+      null;
+  }
+
+  renderTextDescription(text: string): React$Element<*> {
+    return this.renderDescription(text, 'description');
+  }
+
+  renderBoxedDescription(text: string): React$Element<*> {
+    return this.renderDescription(text, 'boxed description');
+  }
```

Comment:
this is nice! if you wanted to make this a little more functional you could do this:

```js
renderDescription = R.curry((className, text) => (
  text.length > 0 ? <div className={className}>{text}</div> : null
));

renderTextDescription = renderDescription('description');

renderBoxedDescription = renderDescription('boxed description');
```

just a thought!